mreg_cli/commands/host_submodules/a_aaaa.py

```diff
@@ -182,7 +183,8 @@ def _add_ip(
 
     mac = None
     if args.macaddress:
-        mac = MACAddressField.validate(args.macaddress)
+        # FIXME: should this raise exception if validation fails? Surely?
+        mac = MacAddress.parse(args.macaddress)
```

Comment:
This needs to be looked at

Reply:
I just changed this to `MacAddress.parse_or_raise`, because it makes no sense to silently fail here, even though we did that before this PR.